.gitignore

```
@@ -1 +1,3 @@
 .vscode/settings.json
+dist/*.whl
+tests/__pycache__
```

```suggestion
__pycache__/
```

This captures all of the pycache folders (pretty standard piece of the Python .gitignores)